src/main/java/bio/terra/command/DatasetCommands.java

```diff
@@ -465,4 +498,50 @@ private static void datasetFileLoad(String datasetName,
             CommandUtils.printErrorAndExit("Error decoding gspath into target URI:\n" + e.getMessage());
         }
     }
+
+    private static void datasetFileLookup(String datasetName,
+                                        String filePath,
+                                        String depth,
+                                        String format) {
+        Integer depthInt = 0;
+        if (depth == null) {
+            depthInt = -1;
+        } else {
+            try {
+                depthInt = Integer.valueOf(depth);
```

Comment:
What happens if someone passes a number less than -1? We could validate here or leave it up to the API.

Reply:
Good point. Added validation here.